lib/kickplan/adapters/http.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Kickplan
+  module Adapters
+    class HTTP < Adapter
+      extend Forwardable
+
+      delegate %i(get post put delete) => :connection
+
+      def connection
+        memoize do
+          Faraday::Connection.new(
+            url: config.endpoint,
+            proxy: config.proxy,
+            builder: config.middleware,
+            headers: {
+              user_agent: config.user_agent
+            }
+          )
+        end
+      end
+
+      def resolve(key = nil, params = {})
+        if key.nil? || key.is_a?(Hash)
+          key, params = nil, key
+        end
+
+        post("features/#{key}", params).body
+      end
```

Comment:
I'm not sure I _love_ using the same method for both resolving a single feature and resolving all features (I really miss elixir's method overloading).

I'll keep it for now, but this might get annoying in the other adapters.

Reply:
Maybe I'll make the adapter interface `resolve` and `resolve_all` and do this differentiation in the resource.